build fails on windows

```

  22) Compiler Tests (HTML)
       use
         6. Generates output for 'using async use classes' correctly.:
     Error: Cannot find module '.\..\specs\use_spec\test_page_async.js'
      at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
      at Function.Module._load (internal/modules/cjs/loader.js:562:25)
      at Module.require (internal/modules/cjs/loader.js:690:17)
      at require (internal/modules/cjs/helpers.js:25:18)
      at main (test\generated\use_6.js:20:18)
```

--- a/.github/workflows/semantic-release.yaml
+++ b/.github/workflows/semantic-release.yaml
@@ -17,23 +17,22 @@ jobs:
       - run: npm install
       - run: npm test
 
-#  build_win:
-#    runs-on: windows-latest
-#    if: "!contains(github.event.head_commit.message, '[skip ci]')"
-#    steps:
-#      - run: git config --global core.autocrlf false
-#      - uses: actions/checkout@v2
-#      - name: Use Node.js 10.x
-#        uses: actions/setup-node@v1
-#        with:
-#          node-version: '10.x'
-#      - run: npm install
-#      - run: npm test
+  build_win:
+    runs-on: windows-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - run: git config --global core.autocrlf false
+      - uses: actions/checkout@v2
+      - name: Use Node.js 10.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+      - run: npm install
+      - run: npm test
 
   release:
     runs-on: ubuntu-latest
-#    needs: [build, build_win]
-    needs: [build]
+    needs: [build, build_win]
     if: "!contains(github.event.head_commit.message, '[skip ci]')"
     steps:
       - uses: actions/checkout@v2

--- a/src/compiler/Compiler.js
+++ b/src/compiler/Compiler.js
@@ -47,6 +47,11 @@ module.exports = class Compiler {
     if (path.isAbsolute(moduleId)) {
       // eslint-disable-next-line no-param-reassign
       moduleId = `.${path.sep}${path.relative(baseDir, moduleId)}`;
+      if (path.sep === '\\') {
+        // nodejs on windows doesn't like relative paths with windows path separators
+        // eslint-disable-next-line no-param-reassign
+        moduleId = moduleId.replace(/\\/, '/');
+      }
     }
     return `const ${varName} = require(${JSON.stringify(moduleId)});`;
   }

